scripts/quests/windurst/Carbuncle_Debacle.lua

```diff
@@ -0,0 +1,187 @@
+------------------------------------
+-- Carbuncle Debacle
+-- !addquest 2 83
+-- Koru-moru !pos -120 -6 124 239
+--
+------------------------------------
+require('scripts/globals/items')
+require('scripts/globals/npc_util')
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/status')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/titles')
+-----------------------------------------------
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.CARBUNCLE_DEBACLE)
+--------------------------------------------------------------------------------------
+
+quest.reward =
+{
+    item            =   xi.items.EVOKERS_HORN,
+    fame            =   60,
+    fameArea        =   xi.quest.fame_area.WINDURST,
+}
+
+quest.sections =
+{
+    --Section: Quest Available
+
+    {
+
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= xi.settings.main.AF3_QUEST_LEVEL and
+                player:getMainJob() == xi.job.SMN and
+                not quest:getMustZone(player)
```

Comment:
check for previous quest complete also. otherwise they will be able to start this one right away.